projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-link/components/linked-cases-table/linked-cases-from-table.component.ts

```diff
@@ -91,6 +94,17 @@ export class LinkedCasesFromTableComponent implements OnInit, AfterViewInit {
       );
   }
 
+  setLinkiedCasesFrom(linkedCases: CaseLinkResponse[]) {
+    const caseLinks = linkedCases.map(item => {
+      return {
+        caseReference: item.caseReference,
+        caseService: item.ccdJurisdiction,
+        caseType: item.ccdCaseType,
+      } as CaseLink
+    });
+    this.linkedCasesService.linkedCasesFrom = caseLinks;
+  }
+
   public getLinkedCases(): Observable<LinkedCasesResponse> {
```

Comment:
Code looks good. Could we have a few test please?

Also is `setLinkiedCasesFrom` a public or private method?

Reply:
@GaryScottHMCTS Tests fixed. Could you please review again?